<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pqExpressionsManagerDialog</class>
 <widget class="QWidget" name="pqExpressionsManagerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>916</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Choose Expression</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>4</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="pqSearchBox" name="searchBox" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>10</height>
         </size>
        </property>
        <property name="settingKey" stdset="0">
         <string notr="true"/>
        </property>
        <property name="advancedSearchEnabled" stdset="0">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="groupChooser"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1" rowspan="3">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="useCurrent">
       <property name="text">
        <string>Use current</string>
       </property>
       <property name="toolTip">
        <string>Use expression in current filter</string>
       </property>
      </widget>
      </item>
      <item>
      <widget class="QPushButton" name="add">
       <property name="text">
        <string>Add</string>
       </property>
       <property name="toolTip">
        <string>Add a new expression</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="remove">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Remove selected expressions</string>
       </property>
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
      </item>
      <item>
      <widget class="QPushButton" name="removeAll">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="toolTip">
        <string>Remove all expressions</string>
       </property>
       <property name="text">
        <string>Remove All</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="importExpressions">
       <property name="toolTip">
        <string>Import expressions from a json file</string>
       </property>
       <property name="text">
        <string>Import</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exportExpressions">
       <property name="toolTip">
        <string>Export expressions to a json file</string>
       </property>
       <property name="text">
        <string>Export</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="toolTip">
        <string>Close without saving</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="save">
       <property name="text">
        <string>Save</string>
       </property>
       <property name="toolTip">
        <string>Save to settings</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="close">
       <property name="text">
        <string>Close</string>
       </property>
       <property name="toolTip">
        <string>Save and close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="pqExpandableTableView" name="expressions">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>true</bool>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>false</bool>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>10</number>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>pqSearchBox</class>
   <extends>QWidget</extends>
   <header>pqSearchBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>pqExpandableTableView</class>
   <extends>QTableView</extends>
   <header>pqExpandableTableView.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../pqComponents.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>close</sender>
   <signal>clicked()</signal>
   <receiver>pqExpressionsManagerDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>368</x>
     <y>527</y>
    </hint>
    <hint type="destinationlabel">
     <x>223</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
